src/pyedb/dotnet/edb.py

```diff
@@ -3682,6 +3685,30 @@ def create_raptorx_setup(self, name=None):
             self.logger.error("RaptorX simulation only supported with Ansys release 2024R2 and higher")
             return False
 
+    def create_hfsspi_setup(self, name=None):
+        """Create an HFSS PI simulation setup from a template.
+
+        Parameters
+        ----------
+        name : str, optional
+            Setup name.
+
+        Returns
+        -------
+        :class:`legacy.edb_core.edb_data.hfss_pi_simulation_setup_data.RaptorXSimulationSetup`
```

Comment:
Can you add that it returns a boolean if something is wrong ?

Reply:
Done